conf/nginx.conf

```diff
@@ -14,14 +14,13 @@ location @app_upstream {
 
 location __PATH__/ {
 
-  root /var/www/bonfire/_build/prod/rel/bonfire/lib/bonfire-0.9.8-classic-beta.26/priv/static;
```

Comment:
I have fuzzy memories of my first testing that lead me to wonder if it won't break the app to change the folder naming... this might need testing.

Reply:
This path (L.17) is not found anywhere in the downloaded file